client/my-sites/earn/components/add-edit-coupon-modal/test/index.jsx

```diff
@@ -0,0 +1,353 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const mockUseTranslate = () => ( text ) => text;
+const mockTranslate = ( text ) => text;
+const mockGetConnectedAccountDefaultCurrencyForSiteId = () => 'USD';
+const mockGetConnectedAccountMinimumCurrencyForSiteId = () => 0.5;
+const RealDate = Date;
+const mockDate = ( isoDate ) =>
```

Comment:
Thank you for this. I needed these for snapshots but then refactored the snapshots out of this file yesterday. This allows me to remove the mockdates from this file entirely. I will probably need to use this for the CouponList component instead though, at least because they use snapshots currently.